backend/src/service/s3Client-service.ts

```diff
@@ -228,6 +223,52 @@ export default class S3ClientService {
         }
     }
 
+    async uploadPSPToS3(
+        buffer: Buffer,
+        targetFilePath: string,
+        transferFolderPath: string
+    ) {
+        console.log("--------->targetPdfFilePath=" + targetFilePath);
+        await this.createFolder(transferFolderPath);
+
+
+        try {
+            const uploadFilecommand = new PutObjectCommand({
+                Bucket: 'dats-bucket-psp',
```

Comment:
Is this supposed to be hard coded?

Reply:
It is temporally , when SMB will be ok , we removed it